puppet/zulip/manifests/base.pp

```diff
@@ -4,16 +4,17 @@
     'debian': {
       $release_name = $::operatingsystemrelease ? {
         # Debian releases
-        /^7.[0-9]*/ => 'wheezy',
-        /^8.[0-9]*/ => 'jessie',
-        /^9.[0-9]*/ => 'stretch',
+        /^7.[0-9]*/  => 'wheezy',
+        /^8.[0-9]*/  => 'jessie',
+        /^9.[0-9]*/  => 'stretch',
+        /^10.[0-9]*/ => 'buster',
```

Comment:
That's a legacy fact; not sure we should use it: https://puppet.com/docs/facter/3.9/core_facts.html#lsbdistcodename

Reply:
The `$::operatingsystemrelease` fact being tested here is also a legacy fact.

Reply:
Then for now I'll just go with the `/^7\.[0-9]*$/` regex pattern since it's the most straightforward approach (rather than worry about using legacy facts).